spec/resources/workbooks_spec.rb

```diff
@@ -82,7 +90,7 @@ def find_or_publish_workbook(name)
     it 'can add group permissions to a workbook' do
```

Comment:
Looks like a partial refactor here. I'm guessing you meant to replace this with the preceding test.